lib/cadence/configuration.rb

```diff
@@ -32,5 +35,22 @@ def initialize
     def timeouts=(new_timeouts)
       @timeouts = DEFAULT_TIMEOUTS.merge(new_timeouts)
     end
+
+    def for_connection
+      Connection.new(
+        type: connection_type,
+        host: host,
+        port: port
+      ).freeze
+    end
+
+    def for_execution
```

Comment:
This seems to only be used for passing in defaults to `ExecutionOptions.new`. Should this be named `default_execution_options` instead?